index.html

```diff
@@ -24,135 +24,14 @@
           </p>
         </div>
         <div class="clmn">
-          <h3>HTML</h3>
+          <h3>Archive</h3>
           <p>
-            Budha is rumored to have invented HTML.
+            Slides from previous courses
           </p>
           <p>
-            <a class="btn" href="html_basics">Take me there</a>
+            <a class="btn" href="archive">Take me there</a>
```

Comment:
I would suggest other approach - just delete everything, put link to https://github.com/Kottans/js-slides/tree/9ab0fb21964055027697ea3935025b894d576bc7 and call it a day, actually. Not all slides were linked from main page in any case.